src/isValidQuery.js

```diff
@@ -0,0 +1,6 @@
+const minMax = new RegExp(/(\()(?:[a-z][a-z]+)(-)((?:[a-z][a-z]+))?(-)?((?:[a-z][a-z]+))(:)(\s?)(\d+)((?:[a-z][a-z]+))\)/)
```

Comment:
@kjaenicke Can you confirm that the matchMedia API we use doesn't error when an invalid string is passed in?